File: 0058-length-of-last-word/0058-length-of-last-word.java
class Solution {
    public int lengthOfLastWord(String s) {
        
        char[] arr = s.toCharArray();
        
        int num = arr.length, i = num-1;
        
        while (arr[i] == ' ') {
            i--;
        }
        
        int j = i;
        
        
        while (i >= 0 && arr[i] != ' ') {
            i--;
        }
        
        
        
        int k = j-i;
        
        return k;
            
            
    }
}

File: 0058-length-of-last-word/0058-length-of-last-word.py
class Solution:
    def lengthOfLastWord(self, s: str) -> int:
        
        i = len(s) - 1
        result = 0

        if s[i] == " ":
            while i >= 0 and s[i] == " ":
                i -= 1

        while i >= 0:
            if s[i] == " ":
                return result
            result += 1
            i -= 1
        
        return result
